python_modules/dagster/dagster/_daemon/asset_daemon.py

```diff
@@ -124,7 +128,17 @@ def run_iteration(
 
             asset_keys = check.not_none(run_request.asset_selection)
 
-            run = submit_asset_run(run_request, instance, workspace, asset_graph)
+            run = submit_asset_run(
+                run_request._replace(
+                    tags={
+                        **run_request.tags,
+                        AUTO_MATERIALIZE_EVALUATION_ID_TAG: str(new_cursor.evaluation_id),
```

Comment:
not quite the correct name for auto observe but it's hidden. We don't currently show auto observe in the UI but we should.

Reply:
can we just name the tag "asset_evaluation_id"? this would line up with the table name (asset_daemon_asset_evaluations), and be more general